src/components/Button/tests/Button.test.tsx

```diff
@@ -220,6 +220,44 @@ describe('<Button />', () => {
     });
   });
 
+  describe('onMouseEnter()', () => {
+    it('is called when the mouse enters button', () => {
+      const onMouseEnterSpy = jest.fn();
+      const button = mountWithAppProvider(
+        <Button onMouseEnter={onMouseEnterSpy} />,
+      );
+      trigger(button.find('button'), 'onMouseEnter');
+      expect(onMouseEnterSpy).toHaveBeenCalledTimes(1);
+    });
+    it('is called when the mouse enters link', () => {
+      const onMouseEnterSpy = jest.fn();
+      const button = mountWithAppProvider(
+        <Button onMouseEnter={onMouseEnterSpy} url="http://google.com" />,
+      );
+      trigger(button.find(UnstyledLink), 'onMouseEnter');
+      expect(onMouseEnterSpy).toHaveBeenCalledTimes(1);
+    });
+  });
+
+  describe('onTouchEnter()', () => {
+    it('is called when the mouse enters button', () => {
+      const onTouchStartSpy = jest.fn();
+      const button = mountWithAppProvider(
+        <Button onTouchStart={onTouchStartSpy} />,
+      );
+      trigger(button.find('button'), 'onTouchStart');
+      expect(onTouchStartSpy).toHaveBeenCalledTimes(1);
+    });
+    it('is called when the mouse enters link', () => {
+      const onTouchStartSpy = jest.fn();
+      const button = mountWithAppProvider(
+        <Button onTouchStart={onTouchStartSpy} url="http://google.com" />,
+      );
+      trigger(button.find(UnstyledLink), 'onTouchStart');
+      expect(onTouchStartSpy).toHaveBeenCalledTimes(1);
+    });
```

Comment:
How do we feel if we add new lines between each `it()` description?

Also - should we have different test descriptions for tests in `onTouchEnter()` with `onMouseEnter()`?